src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountForm.tag

```diff
@@ -7,6 +7,9 @@
 
 <div class="well well-plain">
     <div>
+        <label class="label-control">Adding a Single Instructor</label>
+    </div><br/>
```

Comment:
Noted. Will do the changes. Is a line break required before the <br> or can it be inline with the </div>